diff --git a/PythonKit/PythonLibrary.swift b/PythonKit/PythonLibrary.swift
index 6149478..e6e43e7 100644
--- a/PythonKit/PythonLibrary.swift
+++ b/PythonKit/PythonLibrary.swift
@@ -101,7 +101,7 @@ extension PythonLibrary {
 #if canImport(Darwin)
     private static var libraryNames = ["Python.framework/Versions/:/Python"]
     private static var libraryPathExtensions = [""]
-    private static var librarySearchPaths = ["", "/opt/homebrew/Frameworks/", "/usr/local/Frameworks/"]
+    private static var librarySearchPaths = [""]
     private static var libraryVersionSeparator = "."
 #elseif os(Linux)
     private static var libraryNames = ["libpython:", "libpython:m"]
